unstructured/file_utils/filetype.py

```diff
@@ -51,7 +51,17 @@
 from unstructured.partition.common.metadata import set_element_hierarchy
 from unstructured.utils import get_call_args_applying_defaults, lazyproperty
 
-LIBMAGIC_AVAILABLE = bool(importlib.util.find_spec("magic"))
+# Is the magic *module* available?
+MAGIC_AVAILABLE = bool(importlib.util.find_spec("magic"))
+
+# Is the libmagic *library* also available?
+LIBMAGIC_AVAILABLE = False
+if MAGIC_AVAILABLE:
+    try:
+        import magic
+        LIBMAGIC_AVAILABLE = True
+    except ImportError:
+        pass
```

Comment:
It might be safe and simpler to keep the original constant name with the new logic. something like...


```suggestion
MAGIC_AVAILABLE: bool
if bool(importlib.util.find_spec("magic")):
    try:
        # This should fail if libmagic not installed:
        import magic
    except ImportError:
        MAGIC_AVAILABLE = False
    else:
        # No failure. Magic is available.
        MAGIC_AVAILABLE = True
else:
    MAGIC_AVAILABLE = False
```